frontend/typescript/index.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    root: true,
+    extends: [
+        '../index.js',
+        'airbnb-typescript',
+    ],
+    plugins: ['@typescript-eslint'],
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaVersion: 2020,
+        project: 'tsconfig.json',
+    },
+    overrides: [
+        {
+            files: ["**/*.{ts,tsx}"],
+            rules: {
+                "complexity": ["error", 10],
+                "@typescript-eslint/explicit-function-return-type": ['error'],
+                "@typescript-eslint/no-explicit-any": ['error'],
```

Comment:
**RC:** Please separate `complexity` rule to match only`tsx`. We override complexity rule only for JSX (tsx) files.